lib/code_generator.rb

```diff
@@ -0,0 +1,5 @@
+module CodeGenerator
```

Comment:
@DavidKnott Great job breaking it out into its own method within the module, so we can reuse anywhere!